app/services/induction/transfer_and_continue_existing_fip.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class Induction::TransferAndContinueExistingFip < BaseService
+  def call
+    check_fip_induction_and_different_school!
+
+    ActiveRecord::Base.transaction do
+      # we haven't already been informed this person is leaving
+      if latest_induction_record.active_induction_status?
+        latest_induction_record.leaving!(end_date)
```

Comment:
Will this make the old school see that the person has left? I did the journey yesterday, went back to the old sits dashboard and they were on the transferring out section of the moving school. Just checking we're intending for that to happen?

Reply:
Yes, we don't have much option really. We don't want to have 2 `active` records, this was discussed with Mili/Nicola but perhaps it's worth double checking incase the point was missed

Reply:
I'll double check, it makes sense not to have two active records. I was just thinking about the transfer out journey. If we updating the induction record here, what would need to happen when that journey happens if this is done here? I've not looked enough into that journey to know.